src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,54 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  position: relative;
+
+  border: 3px dotted black;
+  border-radius: 50%;
+  background-color: white;
+
+  &__center {
+    position: absolute;
+    background-color: orange;
+    height: 20px;
+    width: 20px;
+    border-radius: 50%;
+    z-index: 2;
+  }
+
+  &__hand-sec,
+  &__hand-min {
+    position: absolute;
+    z-index: 1;
+    transform-origin: 50% 100%;
+
+    animation-name: move;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+    animation-fill-mode: backwards;
```

Comment:
would be better to combine it in one animation rule 
https://developer.mozilla.org/en-US/docs/Web/CSS/animation
![image](https://github.com/mate-academy/layout_stop-watch/assets/96265844/b6b324e7-e015-47c7-9187-7b29b3abfa2c)